gulp-tasks/copy.js

```diff
@@ -0,0 +1,56 @@
+// packages
+const fs = require("fs");
+const glob = require("glob");
+const path = require("path");
+
+// config
+const assetsDirs = [
+  {
+    src: "./src/assets/img/",
+    dist: "./dist/img/"
+  }
+];
+
+// make sure paths do not end with slash
+function sanitizePath(filepath) {
+  let sanitizedFilepath = filepath;
+  if (filepath.slice(-1) === "/") {
+    sanitizedFilepath = filepath.slice(0, -1);
+  }
+  return sanitizedFilepath;
+}
+
+// copy assets
+function copyAssets(done) {
+  assetsDirs.forEach(dir => {
+    // src and dist
+    let sourceDir = sanitizePath(dir.src);
+    let distDir = sanitizePath(dir.dist);
+
+    // glob all files
+    let files = glob.sync(`${sourceDir}/**/*`, { nodir: true });
```

Comment:
Isn't necessary, `somedir//a` is the same as `somedir/a`.

Reply:
Does node not have a path class? I would feel better pushing components than interpolating a string.